### PR TITLE
Meshstack: Added elementary meshstack, platforms and OSB marketplace …

### PR DESCRIPTION
**Pull request by meshjoerg**
…docs, cleaned up index page

### CODE REVIEW

File: docs/meshstack.aws.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+id: meshstack.aws
+title: Amazon Web Services
+---
+
+*AWS* is a proprietary public cloud platform provided by Amazon Web Services. Meshstack supports project and user management for AWS to include AWS services into cloud projects managed by Meshstack.
```
Comment (sttomm) [inline docs/meshstack.aws.md]: d: As AWS stands for Amazaon Web Services, should we better just say ' provided by Amazon'?

File: docs/meshstack.index.md [hunk 2]
```diff
@@ -3,7 +3,12 @@ id: meshstack.index
 title: Overview
 ---
 
-Welcome to the Meshstack!
+Welcome to the Meshstack Documentation!
 
-This document is targeted at cloud architects and SREs.
+Meshstack is the name of our technology that powers the Meshcloud solution. This document is targeted at cloud architects and SREs and introduces the components of Meshstack and their interaction.
 
+Meshstack is a modular software platform solution that connects organizational processes like project and access management with cloud technology. It provides a unique administration interface to the organization while replicating configuration into attached cloud platforms.
```
Comment (sttomm) [inline docs/meshstack.index.md]: d: Do you mean 'unified' instead of ' unique' ?

File: docs/meshstack.openshift.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+id: meshstack.openshift
+title: OpenShift
+---
+
+Meshstack supports management of RedHat OpenShift platforms. OpenShift has a [Kubernetes](meshstack.k8s.md) core and provides additional services. It is available in both Open Source flavors as well as enterprise offerings by RedHat.
```
Comment (sttomm) [inline docs/meshstack.openshift.md]: d: Should we also add the short remark, as for AWS, that we support SSO, project replication and user management?

File: docs/meshstack.aws.md (hunk 1 above)
Comment (meshjoerg) [reply docs/meshstack.aws.md]: It's a separate entity: 
image so it is fair to say Amazon Web Services, Inc provides the AWS platform.

File: docs/meshstack.index.md (hunk 2 above)
Comment (meshjoerg) [reply docs/meshstack.index.md]: yes :)

File: docs/meshstack.openshift.md (hunk 3 above)
Comment (meshjoerg) [reply docs/meshstack.openshift.md]: yes, good point

**Pull request merged**